packages/apify/src/proxy_configuration.ts

```diff
@@ -415,8 +435,9 @@ export class ProxyConfiguration extends CoreProxyConfiguration {
     }
 
     /**
-     * Checks if Apify Token is provided in env and gets the password via API and sets it to env
+     * Checks if proxy password is provided in env, if not, fetches it from API using Apify Token
```

Comment:
It does not check anything anymore...

```suggestion
     * Fetch & set the proxy password from Apify API if an Apify token is provided.
```

Reply:
Yeah, my bad. Fixed.